docs/proposals/scheduledjob.md

```diff
@@ -0,0 +1,113 @@
+# ScheduledJob Controller
```

Comment:
"Scheduled" is a highly overloaded term, you may want o merge the notion of "duirnal", periodic, etc. "scheduled" is  overused and the context here is ambiguous to someone wanting to use the object for the 1st time.